util/src/test/java/io/grpc/util/GracefulSwitchLoadBalancerTest.java

```diff
@@ -125,17 +125,17 @@ public void switchTo_handleResolvedAddressesAndNameResolutionErrorForwardedToLat
     helper0.updateBalancingState(READY, picker);
 
     ResolvedAddresses addresses = newFakeAddresses();
-    gracefulSwitchLb.handleResolvedAddresses(addresses);
-    verify(lb0).handleResolvedAddresses(addresses);
+    gracefulSwitchLb.acceptResolvedAddresses(addresses);
```

Comment:
Don't change any of these. These shouldn't have caused warnings because they were already marked `@Deprecated`. They are testing the old code flow and need to remain until it is deleted. When the old flow is deleted, the entire set of "// OLD TESTS" will be outright deleted, as we already have tests below that test acceptResolvedAddresses (and test it more fully, as it checks the return value).